### ISSUE TITLE
oelint.vars.valuequoted reported for inline variable settings

### ISSUE THREAD

**Issue by priv-kweihmann** [priv-kweihmann/oelint-adv#77]
```
EXTRA_OECMAKE += "\ 
                 -DBUILD_TESTS=OFF \
"
```

**Comment by priv-kweihmann** [comment]
It's the trailing whitespace in the first line, which breaks the parser - but this is reported as oelint.spaces.linecont:No spaces after line continuation, so it's good for me
